README.md

```diff
@@ -181,32 +183,34 @@ To remove disambiguity we renamed the outputs produced in the `partition_metadat
 ```
 netcdf partition_metadata_3 {
 dimensions:
- NX = 30 ;
- NY = 24 ;
- P = 3 ;
- T = 2 ;                                              0            12        24
- B = 2 ;                                               ┌──────────────────────►  y
- L = 1 ;                                               │
- R = 1 ;                                               │  ┌─────────┬─────────┐
-group: bounding_boxes {                                │  │         │         │
-   domain_x = 0, 0, 20 ;                               │  │         │         │
-   domain_y = 0, 12, 0 ;                               │  │         │         │
-   domain_extent_x = 20, 20, 10 ;                      │  │    0    │    1    │
-   domain_extent_y = 12, 12, 24 ;                      │  │         │         │
+  NX = 30;
+  NY = 24;
+  P = 3;
+  T = 2;
+  0 12 24 B = 2;
+  ┌──────────────────────► y L = 1;
+  │ R = 1;
+  │  ┌─────────┬─────────┐ group : bounding_boxes {
+    │  │         │         │ domain_x = 0, 0, 20;
+    │  │         │         │ domain_y = 0, 12, 0;
+    │  │         │         │ domain_extent_x = 20, 20, 10;
+    │  │ 0    │ 1    │ domain_extent_y = 12, 12, 24;
+    │  │         │         │
   } // group bounding_boxes                            │  │         │         │
-group: connectivity {                                  │  │         │         │
-   top_neighbors = 0, 0, 2 ;                        20 │  ├─────────┴─────────┤
-   top_neighbor_ids = 0, 1 ;                           │  │                   │
-   top_neighbor_halos = 12, 12 ;                       │  │                   │
-   bottom_neighbors = 1, 1, 0 ;                        │  │         2         │
-   bottom_neighbor_ids = 2, 2 ;                        │  │                   │
-   bottom_neighbor_halos = 12, 12 ;                    │  │                   │
-   left_neighbors = 0, 1, 0 ;                       30 ▼  └───────────────────┘
-   left_neighbor_ids = 0 ;
-   left_neighbor_halos = 20 ;                          x
-   right_neighbors = 1, 0, 0 ;
-   right_neighbor_ids = 1 ;
-   right_neighbor_halos = 20 ;
+group:
+  connectivity {
+    │  │         │         │ top_neighbors = 0, 0, 2;
+    20 │  ├─────────┴─────────┤ top_neighbor_ids = 0, 1;
+    │  │                   │ top_neighbor_halos = 12, 12;
+    │  │                   │ bottom_neighbors = 1, 1, 0;
+    │  │ 2         │ bottom_neighbor_ids = 2, 2;
+    │  │                   │ bottom_neighbor_halos = 12, 12;
+    │  │                   │ left_neighbors = 0, 1, 0;
+    30 ▼  └───────────────────┘ left_neighbor_ids = 0;
+    left_neighbor_halos = 20;
+    x right_neighbors = 1, 0, 0;
+    right_neighbor_ids = 1;
+    right_neighbor_halos = 20;
```

Comment:
I don't think we should format the README with clang-format either.

If we do though, we need to wrap my beautiful ascii art with the following,

```
// clang-format off
...
// clang-format on
```

Reply:
Oops! Fixed with force-push.